### PR TITLE
ci: test and build helm chart with github actions

### LINKED ISSUE #772: migrate CI to github actions

### PR DESCRIPTION
**Pull request by lorenzo-cavazzi**
Migrate from TravisCI to GitHub Actions
WIP

fix #772

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -31,3 +31,54 @@ jobs:
...
+  test-chart:
+    needs: test
+    runs-on: ubuntu-latest
+    if: "!startsWith(github.ref, 'refs/tags/') && github.ref != 'refs/heads/master'"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Test helm chart
```
Comment (rokroskar) [inline .github/workflows/ci.yml:44]: we should refactor this into a generic renku action at some point

Comment (lorenzo-cavazzi) [reply .github/workflows/ci.yml:44]: yeah that would make sense... atm it's done in a slightly different way in every repo

**Review by ciyer — APPROVED** [review]
Great to move this repo to GitHub Actions! 👍

**Review by rokroskar — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (lorenzo-cavazzi) [reply .github/workflows/ci.yml:44]: I created an issue here for that https://github.com/SwissDataScienceCenter/renku/issues/1414

**Pull request merged**